src/index.ts

```diff
@@ -15,10 +16,12 @@ mongoose
     })
     .then(() => {
         console.log('Successfully connected to MongoDB database');
-    
+
         const app = express();
 
+        app.use(express.json());
         app.use('/api', apiRoutes);
+        app.use('/rooms', roomRoutes);
```

Comment:
This puts the room routes at `/rooms`, when we want `/api/rooms`. You should move this line (and the import) to `src/routes/api/index.ts` so that it is under the `/api` router.